polyglot/upload.py

```diff
@@ -142,17 +144,17 @@ def get_largest_utxo(self):
 
     def get_split_outputs(self, utxo):
         """(crudely) splits a utxo into 0.001 amounts with some remainder for fees"""
-        num_splits = utxo.amount // 100000 - 1
+        num_splits = int(utxo.amount / (self.fee * 100000)) - 1
```

Comment:
I don't think this is right...

I think it should remain as:
`num_splits = utxo.amount // 100000 - 1`


Otherwise if fee is 0.5 sat / byte `(self.fee * 100000)` evaluates to `50_000`

So number of splits will be calculated for utxo.amount of 50_000 now... but it is supposed to remain at 100_000

Reply:
Also what if self.fee = None?

This function would fail trying to multiple `None * int`